<!DOCTYPE html>
<html lang="en-us">
<head>

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="generator" content="Source Themes Academic 2.4.0">
  <meta name="generator" content="Hugo 0.46" />
  <meta name="author" content="Anh H. Reynolds">
  <meta name="description" content="Physical/Analytical Chemist">
  <link rel="alternate" hreflang="en-us" href="/">
  <meta name="theme-color" content="#0095eb">
  
    
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha512-6MXa8B6uaO18Hid6blRMetEIoPqHf7Ux1tnyIQdpt9qI5OACx7C+O3IVTr98vwGnlcg0LOLa02i9Y1HpVhlfiw==" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.8.6/css/academicons.min.css" integrity="sha256-uFVgMKfistnJAfoCUQigIl+JfUaP47GrRKjf6CTPVmw=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" integrity="sha512-SfTiTlX6kk+qitfevl/7LibUOeJWlt9rbyDn92a1DqWOw9vWG2MFoays0sgObmWazO5BQPiFucnnEAjpAB+/Sw==" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.css" integrity="sha256-ygkqlh3CYSUri3LhQxzdcm0n1EQvH2Y+U5S2idbLtxs=" crossorigin="anonymous">

          <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css" crossorigin="anonymous">
  <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Montserrat:400,700%7cRoboto:400,400italic,700%7cRoboto&#43;Mono">
  <link rel="stylesheet" href="../css/styles.css">
  
  <script type="text/javascript">
  !function(){var analytics=window.analytics=window.analytics||[];if(!analytics.initialize)if(analytics.invoked)window.console&&console.error&&console.error("Segment snippet included twice.");else{analytics.invoked=!0;analytics.methods=["trackSubmit","trackClick","trackLink","trackForm","pageview","identify","reset","group","track","ready","alias","debug","page","once","off","on"];analytics.factory=function(t){return function(){var e=Array.prototype.slice.call(arguments);e.unshift(t);analytics.push(e);return analytics}};for(var t=0;t<analytics.methods.length;t++){var e=analytics.methods[t];analytics[e]=analytics.factory(e)}analytics.load=function(t,e){var n=document.createElement("script");n.type="text/javascript";n.async=!0;n.src="https://cdn.segment.com/analytics.js/v1/"+t+"/analytics.min.js";var a=document.getElementsByTagName("script")[0];a.parentNode.insertBefore(n,a);analytics._loadOptions=e};analytics.SNIPPET_VERSION="4.1.0";
  analytics.load("YOUR_WRITE_KEY");
  analytics.page();
  }}();
</script>
    <script>
      window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
      ga('create', 'UA-126274462-1', 'auto');
      
      ga('require', 'eventTracker');
      ga('require', 'outboundLinkTracker');
      ga('require', 'urlChangeTracker');
      ga('send', 'pageview');
    </script>
    <script async src="//www.google-analytics.com/analytics.js"></script>
    
    <script async src="https://cdnjs.cloudflare.com/ajax/libs/autotrack/2.4.1/autotrack.js" integrity="sha512-HUmooslVKj4m6OBu0OgzjXXr+QuFYy/k7eLI5jdeEy/F4RSgMn6XRWRGkFi5IFaFgy7uFTkegp3Z0XnJf3Jq+g==" crossorigin="anonymous"></script>
    
  <link rel="alternate" href="/index.xml" type="application/rss+xml" title="Anh H. Reynolds">
  <link rel="feed" href="/index.xml" type="application/rss+xml" title="Anh H. Reynolds">
  <link rel="manifest" href="/site.webmanifest">
  <link rel="icon" type="image/png" href="img/icon.png">
  <link rel="apple-touch-icon" type="image/png" href="img/icon-192.png">
  <link rel="canonical" href="/">
  <meta property="twitter:card" content="summary_large_image">
  <meta property="og:site_name" content="Anh H. Reynolds">
  <meta property="og:url" content="/">
  <meta property="og:title" content="Anh H. Reynolds">
  <meta property="og:description" content="Physical/Analytical Chemist">
  <meta property="og:locale" content="en-us">
  <meta property="og:updated_time" content="2017-10-15T00:00:00-04:00">
  

  

<title>Anh H. Reynolds</title>
</head>
<body id="top" data-spy="scroll" data-target="#navbar-main" data-offset="71" >

<nav class="navbar navbar-default navbar-fixed-top" id="navbar-main">
  <div class="container">
    
    <div class="navbar-header">
      
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse"
              data-target=".navbar-collapse" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      
      <a class="navbar-brand" href="../index.html">Anh H. Reynolds</a>
    </div>

    
    <div class="collapse navbar-collapse">
      
      <ul class="nav navbar-nav navbar-right">
        <li class="nav-item">
          <a href="../index.html#about" data-target="#about">
            
            <span>Home</span>
            
          </a>
        </li>

        <li class="nav-item">
          <a href="../index.html#blogs" data-target="#blogs">
            
            <span>Blogs</span>
            
          </a>
        </li>

        <li class="nav-item">
          <a href="../index.html#contact" data-target="#contact">
            
            <span>Contact</span>
            
          </a>
        </li>
      </ul>

    </div>
  </div>
</nav>



<span id="homepage" style="display: none"></span>
  
  <section id="about" class="home-section">
    <div class="container">

<div class="row" itemprop="author" itemscope itemtype="http://schema.org/Person" itemref="person-email person-address">
  <div class="col-xs-12 col-md-4">
    <div id="profile">

      
      <div class="portrait" style="background-image: url('../img/anh.jpg');"></div>
      <meta itemprop="image" content="../img/anh.jpg">
      

      <div class="portrait-title">
        <h2 itemprop="name">Anh H. Reynolds</h2>
        <h3 itemprop="jobTitle">Physical/Analytical Chemist</h3>
        
        <!--
        <h3 itemprop="worksFor" itemscope itemtype="http://schema.org/Organization">
          <span itemprop="name">Northwestern University</span>
        </h3>
        -->
      </div>

      <link itemprop="url" href="/">
      <ul class="network-icon" aria-hidden="true">
        <li>
          <a itemprop="sameAs" href="https://linkedin.com/in/anhreynolds" target="_blank" rel="noopener">
            <i class="fa fa-linkedin big-icon"></i>
          </a>
        </li>
        
        <!--
        <li>
          <a itemprop="sameAs" href="https://www.instagram.com/anhle911/" target="_blank" rel="noopener">
            <i class="fa fa-instagram big-icon"></i>
          </a>
        </li>
        -->
        
        <li>
          <a itemprop="sameAs" href="https://github.com/haianhle" target="_blank" rel="noopener">
            <i class="fa fa-github big-icon"></i>
          </a>
        </li>
        
        <li>
          <a itemprop="sameAs" href="https://scholar.google.com/citations?user=7xgx7FYAAAAJ&hl=en" target="_blank" rel="noopener">
            <i class="ai ai-google-scholar-square big-icon"></i>
          </a>
        </li>

        <li>
          <a itemprop="sameAs" href="mailto:anh.reynolds@gmail.com" target="_blank" rel="noopener">
            <i class="fa fa-envelope big-icon"></i>
          </a>
        </li>
      </ul>

    </div>
  </div>
  <div class="col-xs-12 col-md-8" itemprop="description">
    
<!-- ************************************ CONTENT ******************************-->
<h1>Challenges in Bottom-up Proteomics</h1>
   </br>
   </br>
   <i>working on this...</i>
   </br>
   </br>
<ol>
<li>Sample preparation</li>
<li>Instrumentation: mass spectrometry</li>
    <ul>
    <li>Resolution: ability to distinguish different m/z</li>
    <li>Mass accuracy</li>
    <li>Centroiding process is used to assign each peak with a single m/z value</li>
    <li>The assigned m/z of the centroided peak may be different from the real m/z of the ion – mass error tolerances</li>
    <li>Detection range: usually 100Da to a few thousands Da as proteins are first enzymatically digested into peptides </li>
    <li>Dynamic range: detect proteins and peptides with vast difference in relative abundance</li>
    <li>Orbitrap typically 5-6 orders of magnitude, while peptide and protein concentrations can span > 10 orders of magnitude in a biological matrix</li>
    <li>Sensitivity – the ability to detect low level protein and peptide signals</li>
    <li>Ionization efficiency: peak intensity or ion current count is not directly proportional to molecule’s abundance in the sample</li>
    <li>Ion source used: ESI or MALDI</li>
    <li>Mass analyzer: LIT, QQQ, TOF, FTICR, Orbitrap</li>
    <li>Fragmentation method: CID/CAD, ETD, HCD, etc.</li>
    </ul>
<li>Proteolysis results in unanticipated cleavage products</li>
    <ul>
    <li>Only a small number of peptides/proteins are recovered and identified</li>
    </ul>
<li>The protein inference problem – multiple proteins can share the same subset of peptides under proteolysis (peptide degeneracy)</li>
 <li>Peptides are not detected</li>
    <ul>
    <li>Issue during protein digestion</li>
    <li>Lost on the column during LC</li>
    <li>Unexpected PTMs on the peptides</li>
    <li>Inefficient ionization</li>
    <li>Inefficient fragmentation during MS</li>
    </ul>
<li>Data processing
<li>Data analysis: 
First each MS/MS spectrum is used by a database to identify a peptide sequence.
Then, the peptides are grouped together to identify the proteins.</li>
<li>Qualitative analysis: identification of proteins present in in complex biological matrix</li>
    <ul>
    <li>Erroneous assignments</li>
    <li>Data variations: proteins might not be reproducibly identified from the same sample?</li>
    <li>The absence of evidence for the presence of a peptide is not necessarily evidence for the absence of the peptide</li>
    </ul>
<li>Throughput</li>
</ol>

</br>

<h2>Challenges in data processing and analysis</h2>



<ol>
<li>Data quality: low quality results in high percentage of spectra being unmatched/unassigned</li>
        <ul>
        <li>Noise: poor fragmentation of the peptide ions, poorly selected precursor ions, etc.</li>
        </ul>
<li>Data processing</li>
    <ul>
    <li>Better peak picking algorithm</li>
    <li>Deconvolution algorithms</li>
        <ul>
        <li>Charge deconvolution: conversion of multiply charged ions to singly charged ions</li>
        <li>Noise deconvolution</li>
        <li>Isotope deconvolution</li>
        </ul>
    </ul>
<li>Large amount of input data with hundreds of thousands of MS/MS spectra generated per sample</li>
        <ul>
        <li>Handling of large dataset: size of data exceeds the scalability of software often resulting in a need to divide the dataset, run in batches, and merge data together in the end</li>
        <li>Data compression: no direct access to individual spectrum</li>
            <ul>
            <li>Lossy vs lossless compression</li>
            </ul>
        <li>Data storage and format</li>
            <ul>
            <li>mzXML and mzData double to triple data file size compared to proprietary raw data (has mzML solved this?)</li>
            <li>mzML is 64-bit encoded – not easily readable to human</li>
                <ul><li>raw data is binary and can break XML structure</li>
                    <li>converting binary to text for readability will increase file size</li>
                </ul>
            </ul>
        </ul>
<li>Database search method</li>
        <ul>
        <li>Database needs to be error free</li>
        <li>Peptide-matching algorithms:</li>
            <ul>
            <li>Accurate simulation of the MS/MS spectrum of a peptide given a specific proteolytic digestion from the peptide sequence</li>
                <ul>
                <li>Type of mass spectrometer</li>
                <li>Annotated spectrum library for easier comparison with spectral library</li>
                <li>Indexed relational database</li>
                </ul>
            <li>Scoring function to accurately access the matching between a peptide and a given MS/MS spectrum</li>
                <ul>
                <li>Accurate retention time prediction and have RT included in the scoring function can help increase identification accuracy</li>
                <li>Combining RT with m/z values from traditional mass fingerprinting method</li>
                </ul>
            <li>Certain modified peptides are not present in database </li>
                <ul>
                <li>PTMs of peptides: especially for a variable PTM where a residue may or may not be modified resulting in exponential growth of search space</li>
                </ul>
            <li>Low quality data due to poor fragmentation of peptide</li>
                <ul>
                    <li>Try different fragmentation methods together </li>
                    <li>Multistage MS \((MS^n)\)</li>
                </ul>
            <li>Multiple precursor ions possible result in a complex composite spectra of fragment ions</li>
            <li>Difficulty/bias when matching low-abundance and/or modified proteins</li>
            <li>Search criteria/parameters: too much flexibility results in too large search space and consequently increasing search time and false-positive IDs.</li>
            <li>Proteotypic peptides: those are most confidently observed during fragmentation of proteins through enzyme digestion or chemical cleavage</li>
                <ul><li>Facilitate search when possible</li></ul>
            <li>Interpretation of result: is the match real? True/False Positive/Negative</li>
                <ul><li>Receiver Operating Characteristic (ROC) plot: Sensitivity vs Specificity</li>
                    <li>Decoy database method (or target-decoy search)</li>
                    <li>Result validation method to correctly estimate false discovery rate</li>
                </ul>
            </ul>
        <li>Protein identification from peptides: sequence tag search</li>
            <ul>
            <li>Extrapolation in the absence/missing of data</li>
            <li>“one hit wonders”: when a protein only has one peptide identified</li>
            <li>Identified peptides are not unique to a protein (protein inference problem)</li>
            <li>Target protein sequence is slightly different from existing protein in the database (different species for example) </li>
            <li>Homology search?</li>
            </ul>
        </ul>
<li>Peptide de novo sequencing algorithms when target proteins/peptides are not present in the database</li>
        <ul>
        <li>Imperfect data</li>
            <ul>
            <li>Incomplete ladders with missing fragment ions</li>
            <li>A lot more peaks than just the N-terminal and C-terminal ion ladders</li>
            </ul>
        <li>The only choice for new peptide that is not present in a database</li>
        <li>Explore multiple fragmentation modes for the same peptide and compare results from de novo sequencing of the different spectra</li>
        </ul>
<li>Correctly identifying the proteins is not the same as providing accurate protein sequence</li>
        <ul>
        <li>The original protein sequence might be slightly different from that in the database due to mutations</li>
        <li>Protein sequencing</li>
            <ul>
            <li>Edman degradation</li>
            <li>Digest target protein with multiple enzymes, measure with MS/MS, followed by de novo sequencing and finally assembly of peptides informatically</li>
            </ul>
        </ul>
</ol>

<h2>References</h2>
    <ol>
  <li>Lill et al., Proteomics in the pharmaceutical and biotechnology industry: a look to the next decade (2021) (https://doi.org/10.1080/14789450.2021.1962300)</li>
  <li>Duncan et al. The pros and cons of peptide centric proteomics (2010)
                    (https://www.nature.com/articles/nbt0710-659)</li>
  <li>Sadygov, Cociorva, and Yates III, Large-scale database searching using tandem mass spectra: Looking up the answer in the back of the book (2004)
                    (https://www.nature.com/articles/nmeth725i)</li>
  <li>Ma, Challenges in Computational Analysis of Mass Spectrometry Data for Proteomics (2010) (https://doi.org/10.1007/s11390-010-9309-1)</li>
                <li>Matthiesen, Methods, algorithms and tools in computational proteomics: A practical point of view (2007) DOI: 10.1002/pmic.200700116</li>
    </ol>


<!-- ************************************ END OF CONTENT ******************************-->


<footer class="site-footer">
  <div class="container">

    

    <p class="powered-by">

      &copy; 2022 &middot; 
      Anh H. Reynolds 
      

    </p>
  </div>
</footer>


<div id="modal" class="modal fade" role="dialog">
  <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header">
        <button type="button" class="close btn-large" data-dismiss="modal">&times;</button>
        <h4 class="modal-title">Cite</h4>
      </div>
      <div>
        <pre><code class="modal-body tex"></code></pre>
      </div>
      <div class="modal-footer">
        <a class="btn btn-primary btn-outline js-copy-cite" href="#" target="_blank">
          <i class="fa fa-copy"></i> Copy
        </a>
        <a class="btn btn-primary btn-outline js-download-cite" href="#" target="_blank">
          <i class="fa fa-download"></i> Download
        </a>
        <div id="modal-error"></div>
      </div>
    </div>
  </div>
</div>

    

    
    
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js" integrity="sha512-3P8rXCuGJdNZOnUx/03c1jOTnMn3rP63nBip5gOP2qmUh5YAdVAvFZ1E+QLZZbC1rtMrQb+mah3AfYW11RUrWA==" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.3/imagesloaded.pkgd.min.js" integrity="sha512-umsR78NN0D23AzgoZ11K7raBD+R6hqKojyBZs1w8WvYlsI+QuKRGBx3LFCwhatzBunCjDuJpDHwxD13sLMbpRA==" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha512-iztkobsvnjKfAtTNdHkGVjAYTrrtlC7mGp/54c40wowO7LhURYl3gVzzcEqGl/qKXQltJ2HwMrdLcNUdo+N/RQ==" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.4/isotope.pkgd.min.js" integrity="sha512-VDBOIlDbuC4VWxGJNmuFRQ0Li0SKkDpmGyuhAG5LTDLd/dJ/S0WMVxriR2Y+CyPL5gzjpN4f/6iqWVBJlht0tQ==" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.js" integrity="sha256-X5PoE3KU5l+JcX+w09p/wHl9AzK333C4hJ2I9S5mD4M=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js" integrity="sha256-/BfiIkHlHoVihZdc6TFuj7MmJ0TWcWsMXkeDFwhi0zw=" crossorigin="anonymous"></script>

    <script src="js/hugo-academic.js"></script>
    <script>hljs.initHighlightingOnLoad();</script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"></script>

  </body>
</html>


