<!DOCTYPE html>
<html lang="en-us">
<head>

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="generator" content="Source Themes Academic 2.4.0">
  <meta name="generator" content="Hugo 0.46" />
  <meta name="author" content="Anh H. Reynolds">
  <meta name="description" content="Physical/Analytical Chemist">
  <link rel="alternate" hreflang="en-us" href="/">
  <meta name="theme-color" content="#0095eb">
  
    
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha512-6MXa8B6uaO18Hid6blRMetEIoPqHf7Ux1tnyIQdpt9qI5OACx7C+O3IVTr98vwGnlcg0LOLa02i9Y1HpVhlfiw==" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.8.6/css/academicons.min.css" integrity="sha256-uFVgMKfistnJAfoCUQigIl+JfUaP47GrRKjf6CTPVmw=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" integrity="sha512-SfTiTlX6kk+qitfevl/7LibUOeJWlt9rbyDn92a1DqWOw9vWG2MFoays0sgObmWazO5BQPiFucnnEAjpAB+/Sw==" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.css" integrity="sha256-ygkqlh3CYSUri3LhQxzdcm0n1EQvH2Y+U5S2idbLtxs=" crossorigin="anonymous">

          <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css" crossorigin="anonymous">
  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Montserrat:400,700%7cRoboto:400,400italic,700%7cRoboto&#43;Mono">
  <link rel="stylesheet" href="../css/styles.css">
  
  <script type="text/javascript">
  !function(){var analytics=window.analytics=window.analytics||[];if(!analytics.initialize)if(analytics.invoked)window.console&&console.error&&console.error("Segment snippet included twice.");else{analytics.invoked=!0;analytics.methods=["trackSubmit","trackClick","trackLink","trackForm","pageview","identify","reset","group","track","ready","alias","debug","page","once","off","on"];analytics.factory=function(t){return function(){var e=Array.prototype.slice.call(arguments);e.unshift(t);analytics.push(e);return analytics}};for(var t=0;t<analytics.methods.length;t++){var e=analytics.methods[t];analytics[e]=analytics.factory(e)}analytics.load=function(t,e){var n=document.createElement("script");n.type="text/javascript";n.async=!0;n.src="https://cdn.segment.com/analytics.js/v1/"+t+"/analytics.min.js";var a=document.getElementsByTagName("script")[0];a.parentNode.insertBefore(n,a);analytics._loadOptions=e};analytics.SNIPPET_VERSION="4.1.0";
  analytics.load("YOUR_WRITE_KEY");
  analytics.page();
  }}();
</script>
    <script>
      window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
      ga('create', 'UA-126274462-1', 'auto');
      
      ga('require', 'eventTracker');
      ga('require', 'outboundLinkTracker');
      ga('require', 'urlChangeTracker');
      ga('send', 'pageview');
    </script>
    <script async src="https://www.google-analytics.com/analytics.js"></script>
    
    <script async src="https://cdnjs.cloudflare.com/ajax/libs/autotrack/2.4.1/autotrack.js" integrity="sha512-HUmooslVKj4m6OBu0OgzjXXr+QuFYy/k7eLI5jdeEy/F4RSgMn6XRWRGkFi5IFaFgy7uFTkegp3Z0XnJf3Jq+g==" crossorigin="anonymous"></script>
    
  <link rel="alternate" href="/index.xml" type="application/rss+xml" title="Anh H. Reynolds">
  <link rel="feed" href="/index.xml" type="application/rss+xml" title="Anh H. Reynolds">
  <link rel="manifest" href="/site.webmanifest">
  <link rel="icon" type="image/png" href="img/icon.png">
  <link rel="apple-touch-icon" type="image/png" href="img/icon-192.png">
  <link rel="canonical" href="/">
  <meta property="twitter:card" content="summary_large_image">
  <meta property="og:site_name" content="Anh H. Reynolds">
  <meta property="og:url" content="/">
  <meta property="og:title" content="Anh H. Reynolds">
  <meta property="og:description" content="Physical/Analytical Chemist">
  <meta property="og:locale" content="en-us">
  <meta property="og:updated_time" content="2017-10-15T00:00:00-04:00">
  

  

<title>Anh H. Reynolds</title>
</head>
<body id="top" data-spy="scroll" data-target="#navbar-main" data-offset="71" >

<nav class="navbar navbar-default navbar-fixed-top" id="navbar-main">
  <div class="container">
    
    <div class="navbar-header">
      
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse"
              data-target=".navbar-collapse" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      
      <a class="navbar-brand" href="../index.html">Anh H. Reynolds</a>
    </div>

    
    <div class="collapse navbar-collapse">
      
      <ul class="nav navbar-nav navbar-right">
        <li class="nav-item">
          <a href="../index.html#about" data-target="#about">
            
            <span>Home</span>
            
          </a>
        </li>

        <li class="nav-item">
          <a href="../index.html#blogs" data-target="#blogs">
            
            <span>Blogs</span>
            
          </a>
        </li>

        <li class="nav-item">
          <a href="../index.html#contact" data-target="#contact">
            
            <span>Contact</span>
            
          </a>
        </li>
      </ul>

    </div>
  </div>
</nav>



<span id="homepage" style="display: none"></span>
  
  <section id="about" class="home-section">
    <div class="container">

<div class="row" itemprop="author" itemscope itemtype="http://schema.org/Person" itemref="person-email person-address">
  <div class="col-xs-12 col-md-4">
    <div id="profile">

      
      <div class="portrait" style="background-image: url('../img/Reynolds_Anh.jpg');"></div>
      <meta itemprop="image" content="../img/Reynolds_Anh.jpg">
      

      <div class="portrait-title">
        <h2 itemprop="name">Anh H. Reynolds</h2>
        <h3 itemprop="jobTitle">Physical/Analytical Chemist</h3>
        
        <!--
        <h3 itemprop="worksFor" itemscope itemtype="http://schema.org/Organization">
          <span itemprop="name">Northwestern University</span>
        </h3>
        -->
      </div>

      <link itemprop="url" href="/">
      <ul class="network-icon" aria-hidden="true">
        <li>
          <a itemprop="sameAs" href="https://linkedin.com/in/anhreynolds" target="_blank" rel="noopener">
            <i class="fa fa-linkedin big-icon"></i>
          </a>
        </li>
        
        <!--
        <li>
          <a itemprop="sameAs" href="https://www.instagram.com/anhle911/" target="_blank" rel="noopener">
            <i class="fa fa-instagram big-icon"></i>
          </a>
        </li>
        -->
        
        <li>
          <a itemprop="sameAs" href="https://github.com/haianhle" target="_blank" rel="noopener">
            <i class="fa fa-github big-icon"></i>
          </a>
        </li>
        
        <li>
          <a itemprop="sameAs" href="https://scholar.google.com/citations?user=7xgx7FYAAAAJ&hl=en" target="_blank" rel="noopener">
            <i class="ai ai-google-scholar-square big-icon"></i>
          </a>
        </li>

        <li>
          <a itemprop="sameAs" href="mailto:anh.reynolds@gmail.com" target="_blank" rel="noopener">
            <i class="fa fa-envelope big-icon"></i>
          </a>
        </li>
      </ul>

    </div>
  </div>
  <div class="col-xs-12 col-md-8" itemprop="description">
    
<!-- ************************************ CONTENT ******************************-->
<h1>Bottom-up Proteomics: An Overview</h1>
   </br>
   </br>
   <i>working on this...</i>
   </br>
   </br>
        
MS-based proteomics falls into 2 categories: <b>bottom-up</b> proteomics and <b>top-down</b> proteomics.
 n <b>top-down proteomics</b>, intact proteins are analyzed directly using high-resolution mass spectrometry
and the highly charged protein is fragmented to produce an MS/MS spectrum.
 n <b>bottom-up proteomics</b>, also called shot-gun or peptide-specific proteomics, proteins are digested
via chemical cleavage or proteolysis prior to MS analysis.
The goal of bottom-up proteomics is often to identify and/or quantify all proteins and the complete protein sequences
including post-translational modifications, in a complex biological matrix. 

  </br>
  </br>

Given a complex mixture of proteins, each of which is among over a hundred thousand coded proteins
not including possible post-translational modifications,
the identification of the proteins in this mixture is challenging.
The approach used in bottom-up proteomics is through an LC-MS/MS analysis of the peptide mixture
resulting from site-specific cleavage/proteolysis of the original proteins.
</br>
</br>
This solves 2 problems:
<ol>
 <li> Solubility: Peptides are generally more soluble than proteins </li>
 <li> Sensitivity: LCMS can detect peptides at much lower levels than the parent proteins </li>
</ol>
</br>
The resulting MS/MS spectra can be matched with data from a database of 
simulated MS/MS spectra of peptides generated through <i>in silico</i> digestion of proteins.
Protein identification this way is called <b> automated sequence database searching </b>.
Alternatives are isolation and purification of individual proteins followed by residue-by-residue sequencing,
and sequence inference by de novo interpretation of fragment ion spectra or by means of sequence tags.
Stable isotope-labelled peptides can be added for quantitative analysis of specific proteins.
Label-free quantitation can be done with spectral counting and ion current measurement.

<h2> Standard protocol for protein identification</h2>
<ol>
 <li>Digestion of proteins into peptides</li>
 <li>Peptide separation with liquid chromatography</li>
 <li>MS/MS spectra acquisition</li>
 <ul>
 <li>Data-dependent acquisition (DDA) or data-independent acquisition (DIA) </li>
 <li>LC/1D or 2D gel separation might have to be done off-line for complex samples</li>
 <li>Hundreds of thousands of MS/MS spectra are obtained for one sample:
            some are from noises, and low-level proteins may not result in any spectra</li>
 </ul>
 <li>Protein identification through matching with a database or de novo sequencing</li>
 <ul>
  <li>Database search</li>
  <ul><li>Mascot, PEAKS, Sequest, Tandem, Ommsa, Phenyx, etc.</li></ul>
  <li>De novo sequencing: constructing ladders of fragment ions, for example,
   y-ions (or b-ions) for CID MS/MS spectrum.
  The amino acid sequence of the peptide is determined from the mass differences between adjacent peaks in the ladders</li>
  <ul><li>PEAKS, PepNovo, Lutefisk</li>
 </ul>
</ol>

<h2>Assumptions in bottom-up analysis</h2>
<ol>
 <li>Proteolysis by a specific protease reproducibly results in a small number of peptides</li>
 <li>Unique identification of a protein precursor is possible with a small subset of peptides</li>
 <li>This relationship holds whether the protein is purified or in a complex protein mixture</li>
 <li>The target protein and all its variants are in the database</li>
 <li>Peptides resulting from protein cleavage are fully recovered and detected</li>
</ol>


<h2>Practical applications</h2>
<ol>
 <li><b>Protein identification</b> of a biological sample</li>
  <ul><li>Not very useful as comprehensive, consistent, and reproducible results are not possible</li></ul>
 <li><b>Single-analyte assay</b>: differences in protein levels between two or more sample populations</li>
  <ul><li>Method validation</li>
              <ul><li>Accuracy, precision (repeatability, intermediate precision, and reproducibility)</li>
                    <li>Specificity</li>
              <li>LOD, LOQ, linearity and range</li>
       <li>Ruggedness and robustness</li>
   </ul>
     <li><b>Label-free quantification</b>: fast, cost-effective, simplicity</li>
   <ul><li>Samples are analyzed in separate experiments under the same condition</li>
    <ul><li>Digest protein mixture with a protease such as tryptic digestion</li>
                                </ul>
                            <li>Spectral counting or ion-current measurement (chromatographic peak intensity)</li>
    <ul><li>Ion-current measurement: difficult in practice due to presence of
     multiple confounding factors that compromise precision:
     sample preparation, injection volume, retention time, coeluting species,
     temperature and pressure fluctuations, etc.</li>
                                </ul> 
       <li>Identify and map characteristic peaks for the same peptide from different spectral data (peptide features)</li>
        <ul><li>Retention time correction due to LC variations</li>
                <li>Spectra/sequence alignment algorithms (Smith-Waterman)</li>
                </ul>
       <li>Compute protein ratios from averaged peptide ratios</li>
        <ul><li>Intensity splitting when a peptide feature is shared by multiple proteins</li>
     <li>Outlier removal: errors from incorrect peptide identification or
         overlapping of peptide features</li>
        </ul>
   </ul>
     <li><b>Stable isotope labels</b>: improved precision at the expense of higher cost and complexity</li>
   <ul><li>Isotope-Coded Affinity Tags (ICAT)</li>
       <li>Stable Isotope Labeling by Amino Acids in Cell Culture (SILAC)</li>
       <li>Isobaric Tag for Relative and Absolute Quantitation (iTRAQ)</li>
       <li>Relative intensities between the characteristic peaks are used to compute the quantity ratio</li>
                        </ul>
                </ul>
 <li><b>PTM characterization</b>: report post-translational modifications (PTMs) in the target protein</li>
  <ul><li>Most common PTMs: phosphorylation, glycosylation, methylation, acetylation, and acylation</li>
      <li>Modification sites are not defined by the genome, so often not present in protein database</li>
       <ul><li>Searching for all possible PTMs exponentially increase computational complexity</li>
       </ul>
      <li>Proteins with PTMs may have low abundance and not detectable with MS/MS</li>
      <li>Peptides with PTMs can result in too complex spectra</li>
       <ul><li>Phosphorylation</li>
         <ul><li>β-elimination</li></ul>
              <li>Glycosylation </li>
            <ul><li>Glycan has tree structure instead of linear structure like peptide</li>
                       <li>Glycan can have variable structures and mass values depending on the modification sites</li>
                                     <li>Cleave glycans with enzymes then identify structure of released glycans?</li>
                   </ul>
              </ul>
      <li>The existence and extent of PTMs in a given sample is unknown</li>
               </ul>
 <li><b>PTM quantification</b>: what percentage of proteins are modified by a certain variable PTM</li>
 <li><b>Sequencing of non-standard peptides with non-linear sequence</b>: peptides with disulfide bonds or non-ribosomal peptides</li>
</ol>



     <h2>References</h4>
     <ul>
  <li>Lill et al., Proteomics in the pharmaceutical and biotechnology industry: a look to the next decade (2021) (https://doi.org/10.1080/14789450.2021.1962300)</li>
  <li>Duncan et al. The pros and cons of peptide centric proteomics (2010)
                    (https://www.nature.com/articles/nbt0710-659)</li>
  <li>Sadygov, Cociorva, and Yates III, Large-scale database searching using tandem mass spectra: Looking up the answer in the back of the book (2004)
                    (https://www.nature.com/articles/nmeth725i)</li>
  <li>Ma, Challenges in Computational Analysis of Mass Spectrometry Data for Proteomics (2010) (https://doi.org/10.1007/s11390-010-9309-1)</li>
                <li>Matthiesen, Methods, algorithms and tools in computational proteomics: A practical point of view (2007) DOI: 10.1002/pmic.200700116</li>
     </ul>
</div>

<!-- ************************************ END OF CONTENT ******************************-->


<footer class="site-footer">
  <div class="container">


    <p class="powered-by">

      &copy; 2022 &middot; 
      Anh H. Reynolds 

    </p>
  </div>
</footer>


<div id="modal" class="modal fade" role="dialog">
  <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header">
        <button type="button" class="close btn-large" data-dismiss="modal">&times;</button>
        <h4 class="modal-title">Cite</h4>
      </div>
      <div>
        <pre><code class="modal-body tex"></code></pre>
      </div>
      <div class="modal-footer">
        <a class="btn btn-primary btn-outline js-copy-cite" href="#" target="_blank">
          <i class="fa fa-copy"></i> Copy
        </a>
        <a class="btn btn-primary btn-outline js-download-cite" href="#" target="_blank">
          <i class="fa fa-download"></i> Download
        </a>
        <div id="modal-error"></div>
      </div>
    </div>
  </div>
</div>


      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js" integrity="sha512-3P8rXCuGJdNZOnUx/03c1jOTnMn3rP63nBip5gOP2qmUh5YAdVAvFZ1E+QLZZbC1rtMrQb+mah3AfYW11RUrWA==" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.3/imagesloaded.pkgd.min.js" integrity="sha512-umsR78NN0D23AzgoZ11K7raBD+R6hqKojyBZs1w8WvYlsI+QuKRGBx3LFCwhatzBunCjDuJpDHwxD13sLMbpRA==" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha512-iztkobsvnjKfAtTNdHkGVjAYTrrtlC7mGp/54c40wowO7LhURYl3gVzzcEqGl/qKXQltJ2HwMrdLcNUdo+N/RQ==" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.4/isotope.pkgd.min.js" integrity="sha512-VDBOIlDbuC4VWxGJNmuFRQ0Li0SKkDpmGyuhAG5LTDLd/dJ/S0WMVxriR2Y+CyPL5gzjpN4f/6iqWVBJlht0tQ==" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.js" integrity="sha256-X5PoE3KU5l+JcX+w09p/wHl9AzK333C4hJ2I9S5mD4M=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js" integrity="sha256-/BfiIkHlHoVihZdc6TFuj7MmJ0TWcWsMXkeDFwhi0zw=" crossorigin="anonymous"></script>

    <script src="js/hugo-academic.js"></script>
    <script>hljs.initHighlightingOnLoad();</script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"></script>

  </body>
</html>


